rocky/katalogus/views/boefje_setup.py

```diff
@@ -232,3 +254,18 @@ def create_boefje_with_form_data(form_data, plugin_id: str, created: str | None)
         oci_image=form_data["oci_image"],
         oci_arguments=arguments,
     )
+
+
+def get_interval_minutes(interval_number, interval_frequency) -> int | None:
```

Comment:
```suggestion
def get_interval_minutes(interval_number: int, interval_frequency: str) -> int | None:
```

- It would be nicer to throw a `ValueError` on invalid parameter (e.g. unknown `interval_frequency` argument)
- You can use `Literal["minutes", "hours", ...]` to type hint this literal of choices